R/sem.R

```diff
@@ -509,6 +509,8 @@ checkLavaanModel <- function(model, availableVars) {
 
   # add test statistic correction footnote
   test <- lavaan::lavInspect(semResults[[1]], "options")[["test"]]
+  if(length(test) > 1)
+    test <- test[[2]]
```

Comment:
was there a bug?

Reply:
seems like just an update, the `[["options"]][["test"]]` from `lavInspect` now always contains `standard` as the first entry regardless of the settings.

Reply:
that is not what I see when I run this
```r
library(lavaan)
packageVersion("lavaan") # 0.6.13

HS.model <- ' visual  =~ x1 + x2 + x3 
              textual =~ x4 + x5 + x6
              speed   =~ x7 + x8 + x9 '

fit <- cfa(HS.model, data = HolzingerSwineford1939, estimator = "MLMVS")
lavInspect(fit, "options")[["test"]]
```

Reply:
True, but in that case `length(test)` is still 1, so the code does not do anything different to how it was before the change.

The double entries in the list is when you use the `test` argument:

```r
library(lavaan)
packageVersion("lavaan") # 0.6.13

HS.model <- ' visual  =~ x1 + x2 + x3 
              textual =~ x4 + x5 + x6
              speed   =~ x7 + x8 + x9 '

fit <- cfa(HS.model, data = HolzingerSwineford1939, test = "scaled.shifted")
lavInspect(fit, "options")[["test"]]
```

Reply:
you are absolutely right :)